autoload/lsp.vim

```diff
@@ -807,14 +828,7 @@ function! s:on_notification(server_name, id, data, event) abort
     endif
     call lsp#stream(1, l:stream_data) " notify stream before callbacks
 
-    if lsp#client#is_server_instantiated_notification(a:data)
```

Comment:
this could possible be moved to the semantic file via callbag.

https://github.com/prabirshrestha/vim-lsp/blob/3d0fc4072bef08b578d4a4aa4a6f91de571e99e4/autoload/lsp/internal/diagnostics/state.vim#L42-L58